### PR TITLE
Milestone2

### PR DESCRIPTION
**Pull request by JuliCarracedo**
We've implemented the missing 'Categories', 'Latest News', and 'Latest Deals' sections of the TNW replica, implementing many Grid and responsive design tools.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -185,6 +279,431 @@ <h4 class="money-funded">$530M</h4>
         </div>
     </div>
 
+    <div class="categories">
```
Comment (Elbie-em) [inline index.html]: Kindly change this tag to a *section* tag.

File: index.html [hunk 2]
```diff
@@ -185,6 +279,431 @@ <h4 class="money-funded">$530M</h4>
...
+                <li>
+                    <a class="list-link" href="#">CD Project stock down to 20% after Cyberpunk reviews hit the internet</a>
+                </li>
+                <li>
+                    <a class="list-link" href="#">AirBnb to be bigger than eBay after its long awaited IPO</a>
+                </li>
+             </ul>
+
+        </section>
+
```
Comment (Elbie-em) [inline index.html]: *section* tags wrapped in the category section should be changed to *article* tags

**Review by Elbie-em — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle:

Hello @JuliCarracedo and @YuvrajM108, I would like to congratulate you on your effort in this milestone.👏🏿 Great work on meeting almost all requirements. However, there are some changes to be made so that we can have this project approved.

- [x] Correct use of HTML semantics. 
    - There is an incorrect use of the *section* tag, I have included comments in areas this has occurred and offered correction on what tags to use.
    - The highlights *div* area should be a *section*. The inner *section* tags should be *article* tags.
    - The funding *div* area should be a *section*. 

## [OPTIONAL]

- [x] Wrap all *section* tags in your body in a *main* tag

*For more insight regarding semantics, you can visit this [link](https://www.w3schools.com/html/html5_semantic_elements.asp)*

### Please apply the changes requested so that the project can be approved :heavy_check_mark:

**Feel free to contact me if you have any questions @ Elbie Moonga on slack**

## HAPPY CODING 💻

### :bangbang: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this milestone.

**Review by george-shammar — APPROVED** [review]
### **STATUS : APPROVED** :heavy_check_mark: :heavy_check_mark:

### **I really like your project. It looks exactly like the original**. 

### **Thank you for also adopting the suggested changes**. :+1: 

### **Great job**.!!!

### **Congratulations.** :wine_glass: :wine_glass: 

### **You can go ahead and merge this branch**.

### **I wish you good luck in your future projects**.

**Pull request merged**